### PR TITLE
Fix `each child in a list should have a unique "key" prop.'

### PR DESCRIPTION
**Pull request by sunny0826**
Fix the problem discussed in https://github.com/oam-dev/kubevela/pull/834#discussion_r549551292

### CODE REVIEW

**Review by zzxwill — APPROVED** [review]
LGTM

File: dashboard/src/pages/Capability/Components/ShowComponent/index.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ export default ({ name, parameters }: ShowParameters) => {
             responsive: ['md'],
             render: (text, row) => [
```
Comment (majian159) [inline dashboard/src/pages/Capability/Components/ShowComponent/index.tsx:74]: Should not return an array, should return a element

File: dashboard/src/pages/Capability/Components/ShowComponent/index.tsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ export default ({ name, parameters }: ShowParameters) => {
...
               <Space size="small" align="center">
-                {!row.default ? undefined : <Text code>{text}</Text>}
+                {!row.default ? undefined : <Text code>{text.toString()}</Text>}
               </Space>,
             ],
```
Comment (majian159) [inline dashboard/src/pages/Capability/Components/ShowComponent/index.tsx:78]: Should not return an array, should return a element

**Review by majian159 — CHANGES_REQUESTED** [review]
`Table` `render` Should not return an array, should return a element.

File: dashboard/src/pages/Capability/Components/ShowComponent/index.tsx (hunk 1 above)
Comment (sunny0826) [reply dashboard/src/pages/Capability/Components/ShowComponent/index.tsx]: fixed

**Review by sunny0826 — COMMENTED** [review]
fixed

**Review by hongchaodeng — COMMENTED** [review]
lgtm

**Pull request merged**